pkl-core/src/main/java/org/pkl/core/runtime/VmInfo.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.pkl.core.runtime;
+
+public class VmInfo {
+  private VmInfo() {
+    /* no instantiation */
+  }
+
+  /** Pkl core engine version */
+  public static final String PKL_CORE_VERSION = "0.26.0-dev";
```

Comment:
Certain constants moved here so they can be exposed to other code (for example, telling plugins the current version of `pkl-core`).